config/Makefile.msvc

```diff
@@ -95,7 +95,7 @@ SAFE_STRING=false
 BYTECC=cl -nologo -D_CRT_SECURE_NO_DEPRECATE -O2 -Gy- -MD
 
 ### Additional compile-time options for $(BYTECC).  (For static linking.)
-BYTECCCOMPOPTS=-DCAML_NAME_SPACE
+BYTECCCOMPOPTS=-DCAML_NAME_SPACE -D_WIN32_WINNT=0x0600
```

Comment:
So OCaml is now going to target >= Vista? Or am I wrong?

(unfortunately I don't have enough time to look at things properly now)